Bobjayafam/politico
# #163499719 setup server

#### What does this PR do?
set up server environment to develop the API which will be used to power the frontend of the application
#### Description of Task to be completed?
- install eslint as a linting library
- install babel for transpiling ES6 javascript code to ES5
- create a project folder structure
- install express and dotenv
- configure server.js
- install mocha and chai for tests
- write end point for the home route
- test the endpoint for the home route
#### How should this be manually tested?
- clone the GitHub repo
- checkout branch ``ch-setup-server-163499719``
- run npm install from the command line
- cd ``server``
- run ``npm test`` and you should get a passing test
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163499719
#### Screenshots (if appropriate)
